### PR TITLE
Virtual sink info added on advanced_usage.rst

### PR DESCRIPTION
**Pull request by Chaoscontrol**
## Description

Just added a some extra information to make the use of virtual sink easier for basic users. 

It's my first ever PR, and I'm not a dev, so forgive me if the structure or formatting is not adequate. I just wanted to contribute with the piece of info to help others sort the issue out, after my own struggle. There was also several posts on Discord about this topic, so I think it's going to be useful to have this solved in the docs. 

## Type of Change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Dependency update (updates to dependencies)
- [x] Documentation update (changes to documentation)
- [ ] Repository update (changes to repository files, e.g. `.github/...`)

## Checklist
- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated the in code docstring/documentation-blocks for new or existing methods/components

## Branch Updates
LizardByte requires that branches be up-to-date before merging. This means that after any PR is merged, this branch
must be updated before it can be merged. You must also
[Allow edits from maintainers](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/allowing-changes-to-a-pull-request-branch-created-from-a-fork).
- [x] I want maintainers to keep my branch updated

### CODE REVIEW

**Comment by Bobbar** [comment]
For a virtual audio sink in Windows you only need the [VB-Cable](https://vb-audio.com/Cable/index.htm) driver, which I'm assuming is packaged along side VoiceMeeter.

**Comment by Chaoscontrol** [comment]
Unfortunately it doesn't work. I tested it, and it didn't, and other users reported exactly the same in [this thread](https://discord.com/channels/804382334370578482/1028345164411981824) in Discord.

**Comment by Chaoscontrol** [comment]
Double checked. It indeed works with just VB-Cable, as long as you restart the PC first.

**Comment by ReenigneArcher** [comment]
Thanks, this was on my todo list. I also use virtual audio cable.

**Comment by Chaoscontrol** [comment]
> For a virtual audio sink in Windows you only need the [VB-Cable](https://vb-audio.com/Cable/index.htm) driver, which I'm assuming is packaged along side VoiceMeeter.

I amended based on @Bobbar, and tested myself that VB Cable indeed worked fine as long as you restart the PC.

File: docs/source/about/advanced_usage.rst [hunk 1]
```diff
@@ -436,6 +436,9 @@ virtual_sink
 **Description**
    The audio device that's virtual, like Steam Streaming Speakers. This allows Sunshine to stream audio, while muting
    the speakers.
+   
+   If you don't have a secondary audio device to asign here, you need to install something like `VB Cable <https://vb-audio.com/Cable/index.htm>`_.
```
Comment (Bobbar) [inline docs/source/about/advanced_usage.rst:440]: Minor typo: `asign` >> `assign`

**Comment by ReenigneArcher** [comment]
Thank you for the PR. I've added improvements to the virtual sink section in #1000, along with application examples so I will close this one.

**Pull request closed without merging**